### ISSUE TITLE
git clone with submodules asking password 

### ISSUE THREAD

**Issue by meghbhalerao** [TUMcps/human-robot-gym#9]
I run - `git clone --recurse-submodules git@gitlab.lrz.de:cps-rl/human-robot-gym.git` in my terminal and I get the following prompt - 

```
Cloning into 'human-robot-gym'...
git@gitlab.lrz.de's password: 
```

<img width="947" alt="Screenshot 2024-11-15 at 3 40 47 PM" src="https://github.com/user-attachments/assets/60f50729-1230-4e60-b3b0-a8ab085855f8">

but when I do not use recurse submdules i.e. just the following command - 
`git clone git@github.com:TUMcps/human-robot-gym.git`
 it is able to clone.

am I missing something here? It looks like the old and new owner of the repo is different and hence maybe the issue is happenning - could you please confirm this?

thanks!

**Comment by meghbhalerao** [comment]
Edit - `git clone --recurse-submodules git@github.com:TUMcps/human-robot-gym.git` seems to be working, so maybe it is a typo in the readme, but it would be great to confirm this. thanks!

**Comment by JakobThumm** [comment]
The README is incorrect here, indeed.

**Comment by Moj79a** [comment]
hi. this is correct clone process:
`git clone https://github.com/tumcps/human-robot-gym`

**Comment by meghbhalerao** [comment]
No, please see https://github.com/TUMcps/human-robot-gym/issues/9#issuecomment-2479982957 where I mention what works for me.

**Comment by JakobThumm** [comment]
Changed the README.
